### PR TITLE
Extend Report Month

### PR DESCRIPTION
**Pull request by cmboyman**

### CODE REVIEW

File: reports/extend-report-month.js [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+      fnValueFormat: utils.valueFormat,
+      items: [],
+   };
+}
+
+function GetTeamList(AB) {
+   const getTeamObj = AB.objectByID(OBJECT_IDS.MinistryTeam).model();
+
+   if (getTeamObj == null) {
+      console.error("null value");
```
Comment (guyyoo) [inline reports/extend-report-month.js:30]: Do we need logs?

File: reports/extend-report-month.js [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+         })
+         .catch(bad);
+   });
+}
+
+function GetRCList(AB) {
+   const getRCObj = AB.objectByID(OBJECT_IDS.RC_Name).model();
+
+   if (getRCObj == null) {
+      console.error("null value");
```
Comment (guyyoo) [inline reports/extend-report-month.js:51]: Here too.

File: reports/extend-report-month.js [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+         })
+         .catch(bad);
+   });
+}
+
+function GetYearList(AB) {
+   const getYearObj = AB.objectByID(OBJECT_IDS.GetYear).model();
+
+   if (getYearObj == null) {
+      console.error("null value");
```
Comment (guyyoo) [inline reports/extend-report-month.js:72]: Here too.

**Review by guyyoo — APPROVED** [review]

**Comment by guyyoo** [comment]
And there are conflicting files so I think you should not commit `AppBuilder` which is the submodule.

File: reports/extend-report-month.js (hunk 1 above)
Comment (cmboyman) [reply reports/extend-report-month.js:30]: Already fixed

File: reports/extend-report-month.js (hunk 2 above)
Comment (cmboyman) [reply reports/extend-report-month.js:51]: Already fixed

File: reports/extend-report-month.js (hunk 3 above)
Comment (cmboyman) [reply reports/extend-report-month.js:72]: Already fixed

**Pull request merged**